src/etc/inc/shaper.inc

```diff
@@ -5272,11 +5272,20 @@ function filter_generate_dummynet_rules() {
 		if (!is_module_loaded("dummynet.ko")) {
 			mwexec("/sbin/kldload dummynet");
 		}
-		set_sysctl(array(
+		$sysctls = (array(
 				"net.inet.ip.dummynet.io_fast" => "1",
 				"net.inet.ip.dummynet.hash_size" => "256",
 				"net.inet.ip.dummynet.pipe_slot_limit" => $max_qlimit
 		));
+		init_config_arr(array('sysctl', 'item'));
+		if (is_array($config['sysctl']) && is_array($config['sysctl']['item'])) {
```

Comment:
After `init_config_arr()` these will always be arrays, so this test is unnecessary.

Reply:
fixed